lib/documentLoader.js

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {didIo} = require('bedrock-did-io');
+const {documentLoader} = require('bedrock-jsonld-document-loader');
+
+module.exports = async url => {
+  let document;
+  if(url.startsWith('did:')) {
+    document = await didIo.get({did: url, forceConstruct: true});
```

Comment:
If we don't have an issue for this yet, we should have one ... to turn off `forceConstruct` once we can use live v1 DIDs.

Reply:
Issue opened https://github.com/digitalbazaar/bedrock-profile/issues/69